lib/authentication/view/edit_profile_page.dart

```diff
@@ -259,10 +263,16 @@ class _EditProfilePageState extends State<EditProfilePage> {
     final authProvider = Provider.of<AuthProvider>(context);
     final emailDomain = S.of(context).stringEmailDomain;
     final User user = authProvider.currentUserFromCache;
+
+    if (user == null) {
+      // TODO(AdrianMargineanu): Show error page if user is not authenticated
+      return Container();
```

Comment:
If the user is anonymous or is null the edit profile button is not showing.

Reply:
Yeah, but the page can still be accessed when editing the link in the web version, so we still need to make sure it's not broken.